src/vivarium/framework/logging/manager.py

```diff
@@ -38,7 +37,7 @@ def _terminal_logging_not_configured() -> bool:
         # fragile since it depends on a loguru's internals as well as the stability of code
         # paths in vivarium, but both are quite stable at this point, so I think it's pretty,
         # low risk.
-        return 1 not in logger._core.handlers  # type: ignore[attr-defined]
+        return 1 not in loguru.logger._core.handlers  # type: ignore[attr-defined]
```

Comment:
Why was this change needed?

Reply:
Just a generic cleanup (we're already importing all of loguru). And to make it consistent w/ te component.py